packages/capabilities/src/types.ts

```diff
@@ -36,6 +36,11 @@ export interface AccessClaimFailure {
   error: true
 }
 
+export interface AccessConfirmSuccess {
+  delegations: Record<string, Ucanto.ByteView<Ucanto.Delegation>>
```

Comment:
should I make this

```
  delegations: Ucanto.ByteView<Array<Ucanto.Delegation>>
```
?

I did it this way so it is consistent with other times we encode a set of delegations in response (`delegations-response.js` module).

Figured later we can change to another type as same time as response to access/claim. (changing `delegationsResponse.encode/decode` in one place ideally)

Reply:
I would not bother here and now, but generally sounds good.